src/converters/lintConfigs/rules/ruleConverters/no-suspicious-comment.ts

```diff
@@ -0,0 +1,20 @@
+import { RuleConverter } from "../ruleConverter";
+
+export const convertNoSuspiciousComment: RuleConverter = (tslintRule) => {
+    return {
+        rules: [
+            {
+                ...(tslintRule.ruleArguments.length !== 0
+                    ? { notices: ["ESLint's no-warning-comments does not allow an array of terms to ignore."] }
```

Comment:
Ain't that a contradiction of the rule options? I see `terms` being an array of strings that must match.

Reply:
Hmm yeah. How about...